walkthrough/garden_cookiecutter/{{cookiecutter.directory_name}}/{{cookiecutter.short_name}}.py

```diff
@@ -36,48 +28,40 @@ def run(dest_dir: str) -> None:
 
     df = pd.DataFrame(tb_meadow)
 
-    log.info("{{cookiecutter.short_name}}.harmonize_countries")
-    country_mapping = load_country_mapping()
+    log.info("{{cookiecutter.short_name}}.exclude_countries")
     excluded_countries = load_excluded_countries()
-    df = df[~df.country.isin(excluded_countries)]
-    assert df["country"].notnull().all()
-    countries = df["country"].map(country_mapping)
-    if countries.isnull().any():
-        missing_countries = [
-            x for x in df["country"].drop_duplicates() if x not in country_mapping
-        ]
+    df = df.loc[~df.country.isin(excluded_countries)]
+
+    log.info("{{cookiecutter.short_name}}.harmonize_countries")
+    unharmonized_countries = df["country"]
+    df = geo.harmonize_countries(df=df, countries_file=str(N.country_mapping_path))
+
```

Comment:
We should work on `geo.harmonize_countries` to accept `excluded_countries` as an argument (and several other necessary improvements). For the time being, this boilerplate code is something I would move to `shared.py`. In this case, the goal of moving things to `shared.py` is to make the code inside `run` as simple and understandable as possible.

Reply:
> make the code inside run as simple and understandable as possible.

Yeah, you're right. I moved the code to functions and left only high-level calls in `run` method. I didn't go as far as creating `shared.py`, but I might reconsider that in the future.